src/components/gltf-model-plus.js

```diff
@@ -136,10 +136,11 @@ const inflateEntities = function(node, templates, isRoot) {
     z: node.rotation.z * THREE.Math.RAD2DEG
   });
   el.setAttribute("scale", {
-    x: node.scale.x,
-    y: node.scale.y,
-    z: node.scale.z
+    x: node.scale.x * (scale !== undefined ? scale : 1),
```

Comment:
Should we make scale a vector rather than a scalar value so that the non-uniform scale of persisted models can be set?

Reply:
Or maybe we just don't set a scale property and use the normal transform scale to deserialize that data?